crates/icondiffbot2/src/main.rs

```diff
@@ -1,30 +1,26 @@
 mod github_processor;
 mod job_processor;
+mod runner;
 mod sha;
 mod table_builder;
 
-use diffbot_lib::job::{
-    runner::handle_jobs,
-    types::{JobJournal, JobSender},
-};
+use async_mutex::Mutex;
```

Comment:
Question again, why async_mutex instead of tokio's mutex?

Reply:
idb uses actix's runtime now, it doesn't include a mutex

Reply:
wild